plugins/modules/facter.py

```diff
@@ -73,7 +80,11 @@ def main():
         cmd += module.params['arguments']
 
     rc, out, err = module.run_command(cmd, check_rc=True)
-    module.exit_json(**json.loads(out))
+    results = json.loads(out)
+    if module.params['want_facts']:
+        module.exit_json(ansible_facts=dict(facter=results))
```

Comment:
Ah, I see you put them into a namespace. This needs to be documented.

Reply:
I added that to the RETURN block. Happy to adjust if there's a better way to do that.